src/Execution.ts

```diff
@@ -0,0 +1,110 @@
+import { R4 } from '@ahryman40k/ts-fhir-types';
+import { CalculationOptions } from './types/Calculator';
+
+// import { PatientSource } from 'cql-exec-fhir';
+import cql from 'cql-execution';
+import { PatientSource } from 'cql-exec-fhir';
+import { ELM, ELMIdentifier } from './types/ELMTypes';
+import { dumpELMJSONs, dumpObject, dumpVSMap } from './DebugHelper';
+import { parseTimeStringAsUTC, valueSetsForCodeService } from './ValueSetHelper';
+
+export function execute(
+  measureBundle: R4.IBundle,
+  patientBundles: R4.IBundle[],
+  options: CalculationOptions
+): {
+  rawResults?: cql.Results;
+  errorMessage?: string;
+  elmLibraries?: ELM[];
+  mainLibraryName?: string;
+} {
+  // TODO: return type^
```

Comment:
Obsolete TODO?

Reply:
Yeah, looks that way

Reply:
I think I meant it as a question along the lines as "should we externalize the definition to an `interface`".

Reply:
Ah. I thought it was a comment we'd made when we were returning an `any` here before

Reply:
Ah, that makes sense. i think we can move this return type to one of our type declaration files, yeah

Reply:
Yeah, I'd be fine with that change too